app/assets/javascripts/discourse/app/components/watch-read.js

```diff
@@ -1,39 +1,43 @@
-import { bind } from "discourse-common/utils/decorators";
 import Component from "@ember/component";
+import { inject as service } from "@ember/service";
+import { bind } from "discourse-common/utils/decorators";
 import isElementInViewport from "discourse/lib/is-element-in-viewport";
+import { ajax } from "discourse/lib/ajax";
+import { userPath } from "discourse/lib/url";
+
+export default class WatchRead extends Component {
+  @service currentUser;
 
-export default Component.extend({
   didInsertElement() {
-    this._super(...arguments);
-    const currentUser = this.currentUser;
-    if (!currentUser) {
+    super.didInsertElement(...arguments);
+
+    if (!this.currentUser || this.currentUser.read_faq) {
       return;
     }
 
-    const path = this.path;
-    if (path === "faq" || path === "guidelines") {
```

Comment:
Fixes another issue - visiting /rules or /conduct would show you the faq contents, but wouldn't mark the faq as read 😛